tooling/linux_repro_build_compare.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/sh
+# shellcheck disable=SC2006,SC3010,SC2046,SC2166,SC2086
```

Comment:
Slightly prefer having Shellcheck disables near to the line with a comment explaining the why.

Reply:
done